build/ci/evergreen.yml

```diff
@@ -17,11 +17,6 @@ stepback: true
 command_type: system
 pre_error_fails_task: true
 
-# Files that match an ignore-list pattern will not trigger a build, if they're the only modified
-# files in the patch.
-ignore:
-  - "*.md"
-  - "*.txt"
```

Comment:
This was a problem the other day, since we have evergreen required as part of the PR suit it makes README only changes really hard to merge